Routing/Loader/Reader/RestActionReader.php

```diff
@@ -241,16 +239,11 @@ public function read(RestRouteCollection $collection, \ReflectionMethod $method,
                 }
             }
 
-            if (isset($requirements['_method'])) {
-                $methods = explode('|', $requirements['_method']);
-                unset($requirements['_method']);
-            } else {
-                $methods = array();
-            }
+            $methods = explode('|', strtoupper($httpMethod));
 
             // add route to collection
             $route = new Route(
-                $path, $defaults, $requirements, $options, $host, array(), $methods, $condition
+                $path, $defaults, array(), $options, $host, array(), $methods, $condition
```

Comment:
At this point the `$requirements` only contained this:

https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1054/files#diff-4f168cae49074c5f30b4abcccc85c281L186